components/AppText.tsx

```diff
@@ -17,31 +17,34 @@ export type AppTextI = TextProps & {
     | "primaryTitle"
     | "label"
     | "description"
+    | "astericPasswordText"
     | "passwordText"
+    | "errorText"
     | "detail";
   style?: StyleProp<TextStyle>;
 };
 
 export const AppText = ({ text, style, type = "default", ...rest }: AppTextI) => {
-  const [fontsLoaded] = useFonts(loadFonts());
-
   return (
     <>
       <Text
         style={[
-          fontsLoaded && { fontFamily: AppFont.regular },
+          { fontFamily: AppFont.regular },
           type === "default" ? Typography.default : undefined,
```

Comment:
What is this?? Clean this up and remove all manual turnary operators. Use switch or something better!!!